docs/designers-developers/developers/tutorials/block-tutorial/plugin-sidebar.md

```diff
@@ -124,6 +124,7 @@ add_action( 'enqueue_block_assets', 'sidebar_plugin_style_enqueue' );
 
 ## Step 3: an input that updates its content
 
+```js
```

Comment:
Might be good to check that the markdown parser on dot-org supports language-hinted code fences.

Reply:
It looks like it does. [This tutorial](https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/tutorials/block-tutorial/writing-your-first-block-type.md) uses the php code tag and is rendered properly in the [handbook](https://wordpress.org/gutenberg/handbook/designers-developers/developers/tutorials/block-tutorial/writing-your-first-block-type/).